docs/how_to_guides/create_and_edit_expectations/how_to_create_expectations_between_tables.rst

```diff
@@ -0,0 +1,38 @@
+.. _tutorial_create_expectations_between_tables:
+
+Create Expectations Between Tables
+===================================
+
+This tutorial covers the workflow of creating and editing expectations that relate to data stored in different tables or datasets.
+
+Unfortunately, no Expectations natively support cross-table comparisons today (but we hope to in the future!). Consequently, there are two available paths:
+
+1. Use "Check Assets", where you create a new table that joins the two tables; or
```

Comment:
This should be "check assets" (non-caps), unless we're going to create a linkable article in the Reference section.

Reply:
updated